packages/editor/src/components/widgets/combobox/ComboboxWithBrowser.tsx

```diff
@@ -0,0 +1,30 @@
+import './Combobox.css';
+import { ComponentProps } from 'react';
+import { Browser, BrowserType, useBrowser } from '../../../components/browser';
+import { usePath } from '../../../context';
+import Combobox, { ComboboxItem, ComboboxProps } from './Combobox';
+
+export type ComboboxWithBrowserProps<T extends ComboboxItem> = Omit<ComponentProps<'input'>, 'value' | 'onChange'> &
+  ComboboxProps<T> & {
+    browsers: BrowserType[];
+  };

```

Comment:
instead of copy past the comboboxprops, you could also simply extend it with the browser attribute